index.html

```diff
@@ -150,17 +155,130 @@ <h1 class="greet">
+          <div class="work-img">
+            <img src="assets/Snapshoot Portfoliowork4.png">
+          </div>
+        </div>
+        <div class="desktop_projects" id="desktop_3">
+          <div class="work-img">
+            <img src="assets/Snapshoot Portfolio.png">
+          </div>
+          <div class="desktop_info">
+            <div class="project-title">Facebook 360</div>
+            <div class="some-list">
+              <img src="assets/Frame 2facebook.png">
+            </div>
+            <div class="work-info">
+              A daily selection of private and personalised
+              reads ; no accounts
+              or sign-ups required.
+            </div>
+            <div class="langs">
+              <ul class="lang-list">
+                <li class="Html">html</li>
+                <li class="ruby">Ruby on rails</li>
+                <li class="Css">css</li>
+                <li class="js">javascript</li>
+              </ul>
+            </div>
+            <div class="see-project">
+              <button type="button" class="button">See Project</button>
+            </div>
+          </div>
+        </div>
+        <div class="desktop_projects" id="desktop_4">
+          <div class="desktop_info">
+            <div class="project-title">Uber Navigations</div>
+            <div class="some-list">
+              <img src="assets/Frame 2-1.png">
+            </div>
+            <div class="work-info">
+              A daily selection of private and personalised
+              reads ; no accounts
+              or sign-ups required.
+            </div>
+            <div class="langs">
+              <ul class="lang-list">
+                <li class="Html">html</li>
+                <li class="ruby">Ruby on rails</li>
+                <li class="Css">css</li>
+                <li class="js">javascript</li>
+              </ul>
+            </div>
+            <div class="see-project">
+              <button type="button" class="button">See Project</button>
+            </div>
+          </div>
+          <div class="work-img">
+            <img src="assets/Snapshoot Portfoliowrok2.png">
+          </div>
+        </div>
+      </div>
+        <!--------------------------------------------------------------------->
+      <section class="aboutme_container">
+        <div class="grid_item1">
+        <p class="about_me" id="about_me">About<br>Myself</p>
         <p class="intro_2">
-          Hello I'm A software developer! I can help<br>
-          you build a product,feature or website.<br> 
-          Look through some of my experience!If you have a<br>
-          project you need coded.Don't hesitate<br>
+          Hello I'm A software developer! I can help
+          you build a product,feature or website.
+          Look through some of my experience!If you have a
+          project you need coded.Don't hesitate
           to contact me.
         </p>
         <p class="social-text_2">Let's connect</p><br><br>
```

Comment:
- [ ] Please make sure that you adjust the text to match the figma design as suggested [earlier](https://github.com/Rishi-Mishra0704/Portfolio/pull/5#discussion_r1083284580). Right now, your text has `text-transform: capitalize` property. Please make sure that you use it wisely and adjust the text. I know you can fix this one 💪 Right? 😄
- [ ] I can see that you have increased the `font-size` of heading, please make sure that you increase it's `font-weight` also as suggested earlier so that we have a design exactly according to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848&t=gh6dHTyw1laqWDST-0)
- [ ] Also, please make sure that your social media icons have equal gap as suggested earlier. I know you that fix this 💪
- [OPTIONAL] Also, how about using text-transform: uppercase; for LET'S CONNECT button ?

## **_Your design previously 👇_**
![image](https://user-images.githubusercontent.com/106119105/213881336-1ab0c789-4af3-4f93-9c3a-e561db1347e3.png)

## **_Your design now 👇_**
![image](https://user-images.githubusercontent.com/106119105/213881495-75d37f50-1917-4a33-ba82-900beab7597b.png)


## **_Figma design 👇_**
![image](https://user-images.githubusercontent.com/106119105/213881385-944ddc98-c3ed-4474-872b-049c649c0b45.png)